README.md

```diff
@@ -2,13 +2,15 @@
 
 This action asserts the presence of Deephaven tables with specific names within a running Deephaven instance.
 
+> :warning: **This currently only works with Python Deephaven instances. A future update is coming that will handle Groovy Deephaven instances.**
```

Comment:
Why only python instances?  Seems weird.

Reply:
Using the pyclient, you can only run

```
session.run_script("str")
```

on a python instance. Since that is the current implementation of this action, the groovy instances throw errors. Once https://github.com/deephaven/deephaven-core/issues/1480 is resolved we should be good to go